.travis.yml

```diff
@@ -30,10 +30,8 @@ before_install:
 
 # Install packages
 install:
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION --file requirements.txt
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION jupyter
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pyflakes
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pep8
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION scipy matplotlib h5py jupyter
```

Comment:
> The only reason why I was adding conda install --file requirements.txt is so that the requirements get installed with conda instead of pip. (again, see remark below).

why should we mix the two install methods in travis?